source/installation-guide/installing-wazuh-agent/aix/wazuh_agent_sources_aix.rst

```diff
@@ -0,0 +1,125 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. meta:: :description: Wazuh agent sources installation on AIX
+
+.. _wazuh_agent_sources_aix:
+
+AIX from sources
+================
+
+This guide describes how to install the Wazuh agent from source code for AIX. For other operating systems, please check the list: :ref:`Install Wazuh agent <installation_agents>`.
+
+Installing Wazuh agent
+----------------------
+
+.. note:: All the commands described below need to be executed with root user privileges. Since Wazuh 3.5 it is necessary to have internet connection when following this process.
+
+1. Install development tools and compilers.
+
+   1.1 Download the ``wget`` tool.
+
+     .. code-block:: console
+
+        # rpm -Uvh --nodeps http://www.oss4aix.org/download/RPMS/wget/wget-1.19.2-1.aix5.1.ppc.rpm
+
+   1.2  Download the following script
+
+      .. code-block:: console
+
+        # wget https://raw.githubusercontent.com/wazuh/wazuh-packages/master/aix/generate_wazuh_packages.sh --no-check-certificate
+
+      .. note:: If you can't download the script this way, then you should copy it through the scp utility.
+
+  1.3  Install the necessary dependencies using the script.
+
+      .. code-block:: console
+
+        # chmod +x generate_wazuh_packages.sh
+        # ./generate_wazuh_packages.sh -e
+
+      .. note:: This step may take a few minutes.
+
+2. Download the latest version.
+
+     .. code-block:: console
+
+        # wget -O wazuh.tar.gz --no-check-certificate https://api.github.com/repos/wazuh/wazuh/tarball/v3.9.5 && gunzip -c wazuh.tar.gz | tar -xvf -
```

Comment:
Please indicate the latest Wazuh version,(3.11.4).

Reply:
I have updated to the latest Wazuh's version